src/routes/components/AppFooter.tsx

```diff
@@ -133,20 +137,70 @@ export default function AppFooterPage() {
             {/*Component properties table*/}
             <ComponentProperties properties={componentProperties} />
             <ComponentProperties
-              heading="Footer Secondary nav properties"
+              heading="App Footer Nav Section"
               properties={secondaryNavProperties}
             />
-            <ComponentProperties
-              heading="Meta link properties"
-              properties={metaLinkProperties}
-            />
-
+            
+            {language === "angular" && (
+              <ComponentProperties
+                heading="App Footer Meta Section"
+                properties={metaLinkProperties}
+              />
+            )}
 
             {/* Examples*/}
             <h2 id="component-examples" className="hidden" aria-hidden="true">Examples</h2>
 
             <h3 id="component-example-footer-meta">Footer with meta info</h3>
-            <Sandbox fullWidth>
+            <Sandbox skipRender fullWidth>
+              <CodeSnippet
+                lang="typescript"
+                tags="angular"
+                allowCopy={true}
+                code={`
+                <goa-app-footer maxcontentwidth="100%" slot="meta">
```

Comment:
`slot="meta"` shouldn't be here, it should be in the `goa-app-footer-meta-section` tag below

Reply:
Fixed and updated the PR.

![Test-Footer-Playground-CopyCode](https://github.com/GovAlta/ui-components-docs/assets/47701214/4892b202-c761-49d3-a7a2-86864034c6ed)